.script/tests/KqlvalidationsTests/CustomTables/Findings_data_CL.json

```diff
@@ -0,0 +1,125 @@
+{
+    "Name": "BitSightFindingsData",
```

Comment:
Hello @jayeshprajapaticrest the value of entity " name is duplicated with customtable BitSightFindingsData, and this table share the same content with BitSightFindingsData.

Please replace the value of entity " name " with any other unique value

Reply:
@v-prasadboke 
Corrected.